docs/source/conf.py

```diff
@@ -97,7 +97,11 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+try:
+    import qcarchive_sphinx_theme
+    html_theme = 'qcarchive_sphinx_theme'
+except ModuleNotFoundError:
+    html_theme = 'sphinx_rtd_theme'
```

Comment:
packagers will like this. they get touchy about doing a new package just so they can have docs.